src/Features/Core/Portable/AddParameter/AddParameterService.cs

```diff
@@ -269,11 +270,11 @@ public bool HasCascadingDeclarations(IMethodSymbol method)
             }
         }
 
-        private static List<SyntaxTrivia> GetDesiredLeadingIndentation(
+        private static ImmutableArray<SyntaxTrivia> GetDesiredLeadingIndentation(
             SyntaxGenerator generator, ISyntaxFactsService syntaxFacts,
             SyntaxNode node, bool includeLeadingNewLine)
         {
-            var triviaList = new List<SyntaxTrivia>();
+            using var _ = ArrayBuilder<SyntaxTrivia>.GetInstance(out var triviaList);
```

Comment:
📝 This builder is not actually needed. The result will either be a 1- or 2-element array.

Reply:
i'm ok just keeping the logic here.  trying to contort the code to handle those two cases specially doesn't seem valuble.